README.md

```diff
@@ -18,21 +18,25 @@ Questions can be asked by raising an `Issue`.
   click on the clone button (green in colour).
 
 ```
-
   - Using git on your local machine
 
 ```
- git clone https://github.com/eddiejaoude/hacktoberfest-practice.git
+  git clone https://github.com/eddiejaoude/hacktoberfest-practice.git
```

Comment:
Not sure what the extra space is at the beginning of the line is for?

Btw can you update the URL, now I have moved it to our community organisation, you can accept it here..

```suggestion
git clone https://github.com/EddieJaoudeCommunity/hacktoberfest-practice
```

Reply:
well these were for symmetric alignment

Reply:
Ah ok, then there are some code blocks further down that need them also then to be consistent

Reply:
shall i throw an another pr? or is it fine?

Reply:
mine main motive was to just add that command and that seems fine though I think we won't have to bother much about the alignment what are your thoughts on this?

Reply:
I think it is best either to remove the alignment you changed or change them all - but not half 😉

Reply:
ok alright another pr is on the way 🚀

Reply:
you can update this PR by pushing new commits to it if easier